mapper/census.go

```diff
@@ -524,3 +539,56 @@ func generateTruncatePath(path, dimID string, q url.Values) string {
 	}
 	return truncatePath
 }
+
+func getDataLayerJavaScript(analytics map[string]string) (template.JS, error) {
+	jsonStr, err := json.Marshal(analytics)
+	if err != nil {
```

Comment:
Following on from above...
```suggestion
	jsonStr, _ := json.Marshal(analytics)
```